packages/dds/tree/src/test/domains/json/jsonCursor.bench.ts

```diff
@@ -112,6 +138,38 @@ const canada = generateCanada(
         ? undefined
         : [2, 10]);
 
-describe("ITreeCursor", () => {
-    bench("canada", () => canada);
+function* extractCoordinatesFromCanada(cursor: ITreeCursor): Generator<[number, number]> {
+    cursor.down("features" as FieldKey, 0);
+    cursor.down(EmptyKey, 0);
+    cursor.down("geometry" as FieldKey, 0);
```

Comment:
use `brand` instead of `as` for all these. Its more explicit and more type safe.

Might be better to just define already branded strings for these once as part of the "canada" code as they really are part of its schema.

```suggestion
    cursor.down(brand("geometry"), 0);
```